src/robotoff.ts

```diff
@@ -66,8 +66,8 @@ export class Robotoff {
   }
 
   async insightDetail(id: string) {
-    const result = await this.raw.GET("/insights/detail/{id}", {
-      params: { path: { id } },
+    const result = await this.raw.GET("/insights/detail/{insight_id}", {
```

Comment:
```suggestion
    const result = await this.raw.GET("/insights/detail/{id}", {
```
I think, this should be id because insight_id is not defined.

Reply:
It is taken from the server openapi spec. It is not a string interpolation